src/ledger/AccountFrame.cpp

```diff
@@ -103,12 +103,19 @@ AccountFrame::normalize()
 bool
 AccountFrame::isValid()
 {
+    bool res = (mEntry.lastModifiedLedgerSeq <= INT32_MAX);
```

Comment:
Why not check that the account seqNum is less than INT64_MAX?

Reply:
this is ledger sequence number, a 32 bit number

Reply:
Sorry this comment was pretty unclear in retrospect. What I meant was, why not also check that AccountEntry::seqNum (of type SequenceNumber, which is an alias for uint64) is less than INT64_MAX? The check as your wrote it for LedgerEntry::lastModifiedLedgerSeq is indeed correct.